src/editor/codemirror/language-server/signatureHelp.ts

```diff
@@ -77,44 +92,22 @@ const signatureHelpToolTipBaseTheme = EditorView.baseTheme({
   },
 });
 
-const triggerSignatureHelpRequest = async (
```

Comment:
I think I found this hard to pick up again & understand because this function is called from two very different contexts: a CM command (where it's totally fine / expected to view.dispatch) and and the ViewPlugin update call. Doubtless my doing!